### ISSUE TITLE
unable to extract large gzip files

### ISSUE THREAD

**Issue by rakeshbalusa** [nexB/scancode-toolkit#158]
OS : windows 7 64-bit

I'm trying to extract a gzip file (`download URL` - `http://opensuse-mirror-gce-us.opensu.se/distribution/13.2/repo/oss/suse/repodata/070c71c5f39d4c84872070069631f4572f8338a843df2ae8323b6262c1f91284-other.xml.gz`) which is of `size 99.2 MB` and when I try the command `C:\*****\070c71c5f39d4c84872070069631f4572f8338a843df2ae8323b6262c1f91284-other.xml.gz` I get the following error message : 
```
ERROR extracting: C:/Users/nexB/Documents/070c71c5f39d4c84872070069631f4572f8338a843df2ae8323b6262c1f91284-other.xml.gz: ''
Extracting done.
```
The above `error message is not clear` about what the problem is and the possible solution for the above problem is to read the file in blocks of around 1 MB each. When I used the following code snippet in `uncompress` module in the the extraction was successful.
```
 with open(target_location, 'wb') as uncompressed:
            while True:
                chunk = compressed.read(1024*1024)
                if not chunk:
                    break
                uncompressed.write(chunk)
```

**Comment by pombredanne** [comment]
can you open a separate issue for the error message logging?

**Comment by pombredanne** [comment]
There is a patch in #159

**Comment by rakeshbalusa** [comment]
yes, I will create  a new issue in a few minutes.

**Comment by pombredanne** [comment]
Fixed with #159
